src/components/ClaimFilterWasCategorised.js

```diff
@@ -59,16 +60,27 @@ class ClaimFilterWasCategorised extends Component {
     }
 
     _checkAddCategorization(json_ext, prev_json_ext) {
-        var json_exists = (json_ext === null );
-        var valueUpdated = (prev_json_ext['value']['wasCategorized'] != this.state.categorized && this.state.categorized === true);
-        return (json_exists || valueUpdated)
-        
+        const { filters } = this.props;
+        var json_not_exists = (json_ext === null);
+        var prev_claim_param = prev_json_ext['value']
+        var valueExisted = !!prev_claim_param['claim_ai_quality']
+        var valueUpdated = false
+        var wasCategorized = false
+        if (valueExisted) {
+            wasCategorized = prev_claim_param['claim_ai_quality']['was_categorized']
+        }
+        console.log("Was cat: " + wasCategorized)
```

Comment:
I suppose this is for debugging. Please remove.

Reply:
Removed